ci/container/test_component.sh

```diff
@@ -95,8 +96,11 @@ if [[ -z "$GITHUB_ACTIONS" ]]; then
 fi
 
 echo "[INFO] Running Tests: Test result: $WORKSPACE/junit.xml"
+
 if ! ${MOCHA_CMD[@]} "$SOURCE_ROOT/test/**/*.js"; then
     echo "[ERROR] Test failed"
     [[ -f "$WORKSPACE/junit.xml" ]] && cat $WORKSPACE/junit.xml
     exit 1
 fi
+
+npx nyc report --reporter=text
```

Comment:
who will read this output?

Reply:
This is only possibility to watch report in logs.

Reply:
but it's empty now :)